packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -572,11 +572,11 @@ export class AttributesController extends ConsumptionBaseController {
         });
 
         if (!predecessor.isOwnSharedIdentityAttribute(this.identity.address)) {
-            return ValidationResult.error(CoreErrors.attributes.invalidPredecessor("Predecessor is not a valid own shared identity attribute."));
+            return ValidationResult.error(CoreErrors.attributes.predecessorNotValidOwnSharedIdentityAttribute());
```

Comment:
I feel like _is not valid_ is ambiguous and suggests multiple interpretations:
- the attribute is _invalid_, i.e. the predecessor consists of an altogether erroneous payload
- the attribute is not an own shared identity attribute (which is the actual reason for the error here)
- the attribute's `validFrom` or `validTo` values make it in a way "invalid"

I politely suggest you just remove the "Valid" from the error (and add an "is" for legibility) and call it `predecessorIsNotOwnSharedIdentityAttribute`. Same points apply to some of the other errors.